pkg/sql/pgwire/testdata/pgtest/int_size

```diff
@@ -92,6 +93,6 @@ Query {"String": "SELECT * FROM t1"}
 until
 ReadyForQuery
 ----
```

Comment:
Could you add a couple more tests in here that check what happens when a column gets threaded through a few operators?

I'm thinking we should make sure that, for example, something like `SELECT c FROM t JOIN t2 ON ... WHERE ... ORDER BY d` still tracks `c`'s table id and column id despite being munged up by all of these filter and join operators.

Reply:
sure thing!